src/libs/unwrap/kamino/unwrapKamino.ts

```diff
@@ -0,0 +1,58 @@
+/* eslint-disable no-restricted-syntax */
+import { Account, Connection, sendAndConfirmTransaction } from '@solana/web3.js';
+import {
+  assignBlockInfoToTransaction, createTransactionWithExtraBudget, getAssociatedTokenAddressAndData, Kamino,
+} from '@hubbleprotocol/kamino-sdk';
+import { getConfigByCluster } from '@hubbleprotocol/hubble-config';
+
+const cluster = process.env.APP === 'production' ? 'mainnet-beta' : 'devnet';
+
+export const checkAndUnwrapKaminoTokens = async (
+  connection: Connection,
+  payer: Account,
+) => {
+  const kamino = new Kamino(cluster, connection);
+  const config = getConfigByCluster(cluster);
+  for (const strategyPubkey of config.kamino.strategies) {
+    const strategy = await kamino.getStrategyByAddress(strategyPubkey);
+    if (!strategy) {
+      throw Error('Could not fetch strategy from the chain');
```

Comment:
Lets console.error here and move onto next strategy instead of throwing an error here

Reply:
Fixed